### PR TITLE
[Paladin] Prot T28 profile

### PR DESCRIPTION
**Pull request by OrcinusDrake**
Unsure of what to do with trinkets and whether pants or hands or the better off-set piece. Could also re-run sims for theotar vs nadja as I haven't checked that yet.

### CODE REVIEW

**Comment by OrcinusDrake** [comment]
Using dungeon trinks as the raid ones aren't implemented in simc atm. Also they suck. So might change later if they get retuned.

File: profiles/generators/Tier28/T28_Generate_Paladin.simc [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 
 #covenant=necrolord
-#soulbind=plague_deviser_marileth:4,323074/195:9:1/323091/196:9:1/323095/284:9:1/352110
+#soulbind=plague_deviser_marileth:4,323074/195:11:1/323091/196:11:1/323095/284:11:1/352110
 
 #covenant=night_fae
-#soulbind=dreamweaver:2,319217/196:9:1/319210/319191/195:9:1/352786
+#soulbind=dreamweaver:2,319217/196:11:1/319210/319191/195:11:1/352786
 
 save=T27_Paladin_Protection.simc
```
Comment (Melekus) [inline profiles/generators/Tier28/T28_Generate_Paladin.simc]: Don't forget to update the file name to T28

File: profiles/generators/Tier28/T28_Generate_Paladin.simc [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
-legs=Fateforged_Legplates,id=186348,bonus_id=7187/1498
-feet=Greaves_of_Haunting_Ruination,id=186353,bonus_id=7187/1498
-finger1=Sworn_Oath_of_the_Nine,id=186290,enchant=tenet_of_haste,bonus_id=7187/1498,gem_id=173128
-finger2=Tarnished_Insignia_of_QuelThalas,id=186377,enchant=tenet_of_haste,bonus_id=7187/1498,gem_id=173128
-trinket1=Old_Warriors_Soul,id=186438,bonus_id=7187/1498
-trinket2=Titanic_Ocular_Gland,id=186423,bonus_id=7187/1498
-main_hand=The_Devouring_Cold,id=187056,enchant=sinful_revelation,bonus_id=7187/1498
-off_hand=Guard_of_the_Sundered_Defender,id=186417,bonus_id=7187/1498
+head=Luminous_Chevaliers_Casque,id=188933,bonus_id=7187/1498
+neck=Worldkiller_Iris,id=189859,bonus_id=7187/1524,gem_id=173128
```
Comment (Armadk) [inline profiles/generators/Tier28/T28_Generate_Paladin.simc]:
```suggestion
neck=Worldkiller_Iris,id=189859,bonus_id=6935/7187/1524,gem_id=173128
```

File: profiles/generators/Tier28/T28_Generate_Paladin.simc [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
+shoulder=Luminous_Chevaliers_Epaulettes,id=188932,bonus_id=7187/1505
+back=Shroud_of_the_Sires_Chosen,id=189847,bonus_id=7187/1524,enchant=soul_vitality
+chest=Luminous_Chevaliers_Plackart,id=188929,bonus_id=7187/1505,enchant=eternal_stats
+wrist=Veiled_Auctioneers_Vambraces,id=189825,bonus_id=7187/1524
+hands=Luminous_Chevaliers_Gauntlets,id=188928,bonus_id=7187/1498,enchant=eternal_strength
+waist=Shadowghast_Waistguard,id=171418,bonus_id=6648/6649/8125/6716/1588
+legs=Legguards_of_the_Ultimate_Threat,id=189858,bonus_id=7187/1524
+feet=Shadowghast_Sabatons,id=171413,bonus_id=6648/6649/7054/6716/1588
+finger1=Rygelons_Heraldric_Ring,id=189854,bonus_id=7187/1524,enchant=tenet_of_haste,gem_id=173128
```
Comment (Armadk) [inline profiles/generators/Tier28/T28_Generate_Paladin.simc]:
```suggestion
finger1=Rygelons_Heraldric_Ring,id=189854,bonus_id=6935/7187/1524,enchant=tenet_of_haste,gem_id=173128
```

File: profiles/generators/Tier28/T28_Generate_Paladin.simc [hunk 4]
```diff
@@ -1,40 +1,40 @@
...
+finger2=Loquacious_Keepers_Peridot,id=189802,bonus_id=7187/1524,enchant=tenet_of_haste,gem_id=173128
```
Comment (Armadk) [inline profiles/generators/Tier28/T28_Generate_Paladin.simc]:
```suggestion
finger2=Loquacious_Keepers_Peridot,id=189802,bonus_id=6935/7187/1524,enchant=tenet_of_haste,gem_id=173128
```

File: profiles/generators/Tier28/T28_Generate_Paladin.simc (hunk 1 above)
Comment (Armadk) [inline profiles/generators/Tier28/T28_Generate_Paladin.simc]:
```suggestion
save=T28_Paladin_Protection.simc
```

**Review by Armadk — CHANGES_REQUESTED** [review]
bonus_id 6935 should be added for items with sockets

Current rules allow for a socket on head, neck, wrists, waist, and both rings.  So you may want to also include sockets for head, wrist, and waist.

**Comment by OrcinusDrake** [comment]
Okay implemented those suggestions, thank you. Based on newer information, upgraded the hands to 285 too.

**Review by Armadk — APPROVED** [review]

**Pull request merged**